zulipterminal/config/keys.py

```diff
@@ -415,3 +426,19 @@ def commands_for_random_tips() -> List[KeyBinding]:
         for key_binding in KEY_BINDINGS.values()
         if not key_binding.get("excluded_from_random_tips", False)
     ]
+
+
+# Maps alternate nav keys to standard nav keys.
+# Refer urwid/command_map.py
+for cmd in STD_NAV_CMDS:
+    std_key, *alt_keys = keys_for_command(cmd)
```

Comment:
I know that we have the following comment in the file. However, it is not "safe" to assume that the first key would be the `primary/std` key; perhaps we should use `primary_key_for_command` to fetch the `primary/std` key.
```python3
    # Key that is displayed in the UI is determined by the method
    # primary_key_for_command. (Currently the first key in the list)
```

Reply:
`primary_key_for_command` also picks the first key, it's basically the same thing ryt?

Reply:
Well, there could be discrepancies later down the road. The code is currently relying on the _fact_ that the first key is the primary key for now. However, that might change later warranting a manual change in this code block.

Reply:
I agree with Preet; if we later changed the feature a lot, it would also be a lot simpler to look for uses of `primary_key_for_command` than cases like this - and if we decided to make the last key important then we just have to change one function, not the function and this extra position.